x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_process_detail.tsx

```diff
@@ -56,73 +56,42 @@ export const ProcessDetails = memo(function ProcessDetails({
     const dateTime = eventTime ? formatDate(eventTime) : '';
 
     const createdEntry = {
-      title: i18n.translate(
-        'xpack.securitySolution.endpoint.resolver.panel.processDescList.created',
-        {
-          defaultMessage: 'Created',
-        }
-      ),
+      title: '@timestamp',
       description: dateTime,
     };
 
     const pathEntry = {
-      title: i18n.translate('xpack.securitySolution.endpoint.resolver.panel.processDescList.path', {
-        defaultMessage: 'Path',
-      }),
+      title: 'process.executable',
       description: processPath(processEvent),

```

Comment:
double checking, but should the descriptions still be `processPath`

Reply:
Thanks for checking, yeah that's what they call it in ECS:
![image](https://user-images.githubusercontent.com/1087008/87097336-f0994d80-c212-11ea-9175-c0b815c17c4f.png)